enclave-tls/src/tls_wrappers/wolfssl/un_negotiate.c

```diff
@@ -234,8 +234,11 @@ int verify_certificate(int preverify, WOLFSSL_X509_STORE_CTX *store)
     	size_t ext_data_len;
 
 	strncpy(evidence.type, "nullquote", sizeof(evidence.type));
+	/* Essentially speaking, sgx_ecdsa_qve verifier generates the same format of quote as
+	 * sgx_ecdsa.
+	 */
 	if (find_oid(der_cert, der_cert_len, qve_quote_oid, ias_oid_len, &ext_data, &ext_data_len) == 1)
-		strncpy(evidence.type, "sgx_ecdsa_qve", sizeof(evidence.type));
+		strncpy(evidence.type, "sgx_ecdsa", sizeof(evidence.type));
```

Comment:
please modify the following struct .type
static enclave_quote_opts_t sgx_ecdsa_qve_opts = {
	.api_version = ENCLAVE_QUOTE_API_VERSION_DEFAULT,
	.flags = ENCLAVE_QUOTE_FLAGS_DEFAULT,
	**.type = "sgx_ecdsa_qve"**,
	.priority = 53,
	.pre_init = sgx_ecdsa_pre_init,
	.init = sgx_ecdsa_qve_init,
	.collect_evidence = sgx_ecdsa_collect_evidence,
	.verify_evidence = sgx_ecdsa_verify_evidence,
	.cleanup = sgx_ecdsa_cleanup,
};

Reply:
According to current implemention, the cross-verification/attestation will fail( server will select "nullquote" as evidence type)